### ISSUE TITLE
README refers to a nonexistent DOT rule

### ISSUE THREAD

**Issue by selkovjr** [BIO-DIKU/SeqScan#113]
It does so in this tetraloop example:

  seqscan -p 'p1=4..6 .{4} ~p1' -o hits.tsv input.fna

The only rules involving DOT are

INT DOT DOT DOT INT
INT DOT DOT INT

**Comment by maasha** [comment]
Yeah, I am afraid that was never implemented. But IIRC `N{4}` and `4...4` should work.

**Comment by RasmusFonseca** [comment]
I'm not sure why we'd need more than one wildcard notation. Is it ok that I change the tetraloop example to `p1=4..6 4..4 ~p1` for now?

**Comment by maasha** [comment]
The .. and ... notation is a leftover from the original scanformatches. The new notation is a bit more versatile. I don't see a problem in supporting both.
